frontend/packages/topology/src/behavior/withRemoveConnector.tsx

```diff
@@ -10,23 +12,31 @@ type ElementProps = {
 export type WithRemoveConnectorProps = {
   onShowRemoveConnector?: () => void;
   onHideRemoveConnector?: () => void;
+  removeEdgeRef?: (svgElement: SVGElement) => void;
 };
 
 type RemoveRenderer = (
   edge: Edge,
   onRemove: (edge: Edge) => void,
+  startPoint?: Point,
+  endPoint?: Point,
```

Comment:
Probably useful to make these always present with the defaults being the edge start and end point?

Reply:
when using `withRemoveConnector` we don't yet have the edge.

Reply:
Actually, with the rework, this is no longer needed.